.github/workflows/vagrant-box-atom.yml

```diff
@@ -0,0 +1,36 @@
+name: AtoM Vagrant box
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version
+        required: true
+      description:
+        description: Description
+        required: true
```

Comment:
I'd love to see a textarea input in here but it's currently not possible. The easiest solution I found is to add `<br/>` directly  in the input and those are represented in Vagrant Cloud's GUI as expected, but they break the Markdown support. We could try to add some extra formatting in the Ruby script to accept other linebreaks (like `\n`) but I'm not sure if it worths the effort.

Reply:
One potential improvement for the future is to maintain a CHANGELOG.md document (there are many changelog parsers or plain markdown parsers) in this repo that is read by this workflow to extract the description field from there - it's somewhat more reproducible too since the build inputs will be versioned.